wiki/Beatmap_search/de.md

```diff
@@ -117,7 +120,19 @@ unplayed= status=r christmas
 ranked>=2010-08 ranked<2010-11 creator=Natteke
 ```
 
+(Lazer) Finde Beatmaps, die zuletzt vor maximal 2 Monaten und 5 Stunden gespielt wurden:
```

Comment:
Is this simplification in word choice problematic here?

Reply:
> Is this simplification/generalisation in word choice problematic here? (beatmap instead of beatmap difficulty)

I think we should use the correct term. It doesn't require a lot of effort to be precise here AFAICT.

Reply:
```suggestion
(Lazer) Finde Schwierigkeitsstufen von Beatmaps, die zuletzt vor maximal 2 Monaten und 5 Stunden gespielt wurden:
```
*Note: Personally, I prefer this one.*

Reply:
```suggestion
(Lazer) Finde Beatmap-Schwierigkeitsstufen, die zuletzt vor maximal 2 Monaten und 5 Stunden gespielt wurden:
```